src/cases/containers/case-hearings/case-hearings.component.html

```diff
@@ -1,4 +1,4 @@
-<xuilib-loading-spinner *ngIf="showSpinner$ | async"></xuilib-loading-spinner>
```

Comment:
I don't think this is  a good idea. The  loading service is tracking the loading of the service hearing values call, and this is asynchronous. You are now depending on it to complete in a timely way before the spinner is pulled down by the LSH call completing. Probably it will, but we don't have control over how long it takes. If the service is calling CCD to fetch the data it might take a long time sometimes.

It might be better to have 2 observables - the existing showSpinner$ and a new one that emits at the end of ngOnInit, which are combined using combineLatest

Reply:
Hi @andywilkinshmcts , I tried what we proposed on Friday but the problem persisted.  The solution above worked as it was put at the end of the function call which itself was called in the ngOnInit.

I will revisit the ticket to address your concerns though.

Reply:
@andywilkinshmcts , please check my latest fixes. Thanks.